src/wormhole/transit.py

```diff
@@ -596,7 +599,7 @@ def _build_listener(self):
         direct_hints = [
             DirectTCPV1Hint(six.u(addr), portnum, 0.0) for addr in addresses
         ]
-        ep = endpoints.serverFromString(reactor, "tcp:%d" % portnum)
+        ep = endpoints.serverFromString(twisted.internet.reactor, "tcp:%d" % portnum)
```

Comment:
Yeah, you're right, this should really be `self._reactor`. I'll see if I can figure out what was making that test fail.